src/server/pbsd_init.c

```diff
@@ -478,11 +455,8 @@ pbsd_init(int type)
 		}
 #endif	/* RLIMIT_CORE */
 	}
-#endif	/* WIN32 */
 
 #else	/* setrlimit 32 bit */
```

Comment:
I believe we do not need the 32 bit limit setting code, since server (pbsd_init.c is part of server only), is built only in 64 bit mode on all platforms now that windows server is gone